extensions/wikia/ARecoveryEngine/ARecoveryEngineApiController.class.php

```diff
@@ -14,11 +14,19 @@ public function getDelivery() {
 		$this->response->setCacheValidity( WikiaResponse::CACHE_STANDARD );
 	}
 
+	public function getSourcePointStatus() {
+		$this->response->setContentType( 'text/javascript; charset=utf-8' );
+		$wgGlobalEnableSourcePoint = WikiFactory::getVarValueByName( 'wgGlobalEnableSourcePoint', Wikia::COMMUNITY_WIKI_ID );
+		$this->response->setBody( 'window.wikiaSourcePointStatus = ' . ( $wgGlobalEnableSourcePoint ? 'true;' : 'false;' ) );
+		$this->response->setCacheValidity( WikiaResponse::CACHE_STANDARD );
```

Comment:
Check browser caching headers - we may want to add 2nd parameter and do not cache it in user's browsers.